Gestor 100% funcional

carloscmor opened the pull request.

Commit:
Solucionado el error de 30 días

Se seleccionaban cuentas que no tenían operaciones, por lo que se lanzaba error por nulidad de operaciones.

Commit:
Mejoras varias en gestor

-Mejoras de estilos.
-Mejoras en el filtro de clientes.

Commit:
Errores solucionados en RolUsuarioEntity

Commit:
Gertor 100% funcional

La sección de gestor presenta todos los requisitos mínimos implementados

Dani-gr commented on BancApp/src/main/java/es/proyectotaw/banca/bancapp/controller/GestorController.java:
```diff
@@ -235,4 +237,56 @@ public String doMostrarEmpresa(@RequestParam("id") Integer id, Model model, Http
+            case "e" : filtrada = ops.stream().filter(op -> op.getExtraccionsByIdOperacion() != null &&
+                    !op.getExtraccionsByIdOperacion().isEmpty() && op.getExtraccionsByIdOperacion().get(0) != null).toList();
+                    break;
+        }
```
Debería haber una branch default por si acaso imo

Dani-gr commented on BancApp/src/main/webapp/WEB-INF/view/cabeceraMenuGestor.jsp:
Quitemos el comentario xD

Dani-gr commented on BancApp/src/main/webapp/WEB-INF/view/solicitudesDesbloqueoEmpresaGestorView.jsp:
```diff
@@ -32,7 +35,8 @@
                 <td> <%=r.getEmpresaByIdempresa().getCif()%> </td>
-                <td> <a href="/gestor/desbloquearEnEmpresa?rolUsuario= <%=r.getId()%> ">ACTIVAR</a> </td>
+                <td> <%=r.getRolByIdrol().getNombre()%> </td>
+                <td> <a class="btn btn-success" href="/gestor/desbloquearEnEmpresa?rolUsuario= <%=r.getId()%> ">ACTIVAR</a> </td>
```
Por consistencia con el refactor que hiciste el la línea 25, debería ser "DESBLOQUEAR" imo

Commit:
Mejoras

-default switch.
-comentario retirado.
-Desbloquear en vez de activar en personal de la empresa.
